components/scream/tests/scream_homme_dyn_p3/CMakeLists.txt

```diff
@@ -2,36 +2,34 @@ INCLUDE (ScreamUtils)
 
 # Get or create the dynamics lib
 #                 HOMME_TARGET   NP PLEV QSIZE_D USE_PIO USE_ENERGY 
-CreateDynamicsLib("preqx_kokkos"  4   72   35    FALSE   FALSE)
+CreateDynamicsLib("theta-l_kokkos"  4   72   35    FALSE   FALSE)
```

Comment:
Here you keep the USE_PIO and USE_ENERGY columns, but not elsewhere.  Is there a reason why?

Reply:
Ah, this test is not even compiled, so I didn't notice the old signature of the macro. I will fix this, but keep in mind this folder is skipped anyways.